src/Rox/Core/Repositories/TargetGroupRepository.php

```diff
@@ -26,6 +26,10 @@ function setTargetGroups($targetGroups)
      */
     function getTargetGroup($id)
     {
+        if ($this->_targetGroups === null) {
+            throw new \Exception('TargetGroupRepository._targetGroups is not set.');
+        }
```

Comment:
There's probably a better way of going about this, but for whatever reason `$this->_targetGroups ?? []` and a plain ol' `return null` didn't work as expected. I should've documented what was going on more thoroughly when I made the change :sweat_smile: